source/glimmer-components/index.md

```diff
@@ -0,0 +1,37 @@
+## HTML elements as an abstraction for text documents
+
```

Comment:
Can you say more what you mean?

There are certainly three audiences here:

* people familiar with HTML/CSS/JS
* people familiar with other web frameworks that have a component API
* people familiar with the current curly components in Ember

This starts off with a common denominator: all three groups know HTML.